packages/components/src/PhaseCountdown.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import { storiesOf } from "@storybook/react";
+import * as React from "react";
+import styled from "styled-components";
+import { CountdownTimer } from "./PhaseCountdown";
+
+const StyledDiv = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 400px;
+`;
+
+const Container: React.StatelessComponent = ({ children }) => <StyledDiv>{children}</StyledDiv>;
+const now = Date.now() / 1000;
+const eightDaysFromNow = now + 86400 * 8;
```

Comment:
not super important since this is just a story, but probably worth adding some time-manipulation functions in `utils` package for this type of thing